utils/src/ooinstall/openshift_ansible.py

```diff
@@ -175,30 +184,20 @@ def write_proxy_settings(base_inventory):
         pass
 
 
-# pylint: disable=too-many-branches
 def write_host(host, role, inventory, schedulable=None):
     global CFG
 
     if host.preconfigured:
         return
 
     facts = ''
-    if host.ip:
-        facts += ' openshift_ip={}'.format(host.ip)
-    if host.public_ip:
-        facts += ' openshift_public_ip={}'.format(host.public_ip)
-    if host.hostname:
-        facts += ' openshift_hostname={}'.format(host.hostname)
-    if host.public_hostname:
-        facts += ' openshift_public_hostname={}'.format(host.public_hostname)
-    if host.containerized:
-        facts += ' containerized={}'.format(host.containerized)
+    for prop in HOST_VARIABLES_MAP:
```

Comment:
I took a swing at refactoring that stuff into the `Host` class during the code freeze, I got it working 1:1 w/ current behavior (verified via new unit test)

https://github.com/tbielawa/openshift-ansible/commit/3d76b487b098bb30bdf136a7c6ff0ddbab3cb3bc#diff-3e7671fdf1d178a3b0619520545c963eR164

Diff against master: https://github.com/openshift/openshift-ansible/compare/master...tbielawa:RefactorHostClass